homeassistant/components/light/vera.py

```diff
@@ -0,0 +1,181 @@
+"""
+Support for Vera lights.
+
+Configuration:
+This component is useful if you wish for switches connected to your Vera
+controller to appear as lights in homeassistant.  All switches will be added
+as a light unless you exclude them in the config.
+
+To use the Vera lights you will need to add something like the following to
+your config/configuration.yaml
+
+light:
+    platform: vera
+    vera_controller_url: http://YOUR_VERA_IP:3480/
+    device_data:
```

Comment:
If you use a hash instead of a list you can get rid of the `get_extra_device_data` method.

```yaml
light:
  platform: vera
  device_data:
    12:
      name: My Awesome switch
      exclude: true
    13:
      name: Another switch
```